transmogrifier/sources/json/aardvark.py

```diff
@@ -70,6 +91,7 @@ def get_optional_fields(self, source_record: dict) -> dict | None:
         fields["contributors"] = self.get_contributors(source_record) or None
 
         # dates
+        fields["dates"] = self.get_dates(source_record, source_record_id) or None
```

Comment:
Just curious, what is the definition for the `dates` field of a TIMDEX record?

Reply:
The `Date` class is defined [here](https://github.com/MITLibraries/transmogrifier/blob/97b8bc1134f3c19c35870a0766afcc2421e75de2/transmogrifier/models.py#L68). In general, it's for any type of date, with a `kind` qualifier strongly preferred